scripts/variantstore/tieout/AoU_PRECISION_SENSITIVITY.md

```diff
@@ -1,104 +1,82 @@
-```
-mkdir -p truth && gsutil -m cp gs://broad-dsp-spec-ops/gvs/truth/* truth/
-```
-4. Complete (at least) through the training step (`GvsCreateFilterSet`) of the [GVS pipeline](../AOU_DELIVERABLES.md) with all the samples (control and non-control).  Then run the callset extract step with a view of the sample-mapping table that's controls-only but with the training (i.e. "filter_set") from all the samples.
-## Generate gVCFs for Chromosome 20
-Once you have the "controls only" run complete, inspect the shards to find the set of shards from `GvsExtractCallset` that encompass chr20 entirely, e.g. looking at different values of `shard-???` where the output of:
-```
-gsutil cat alpha1_1000_???.vcf.gz | gunzip | grep -v "#" | cut -f1-8 | head
-```
-starts with "chr20". Once the range of shards is known, gather those into a single chr20 gVCF for all samples with something like:
+
+## Prerequisites
+Complete (at least) through the training step (`GvsCreateFilterSet`) of the [GVS pipeline](../AOU_DELIVERABLES.md) with all the samples (control and non-control).  Then run the callset extract step with a view of the sample-mapping table that's controls-only but with the training (i.e. "filter_set") from all the samples.
+
+1. Use the GvsCalculatePrecisionAndSenitiviy wdl to calculate the precision and sensitivity..
+
+The wdl takes several inputs:
+
+**input_vcfs** - A collection of VCFS - generated by `GvsExtractCallSet`. These need not be subsetted down to the chromosome.
+
 ```
-gatk --java-options -Xms6g GatherVcfsCloud --ignore-safety-checks --gather-type BLOCK \
--I aou_callset_1K_223.vcf.gz \
--I aou_callset_1K_224.vcf.gz \
--I aou_callset_1K_225.vcf.gz \
--I aou_callset_1K_226.vcf.gz \
--I aou_callset_1K_227.vcf.gz \
--I aou_callset_1K_228.vcf.gz \
--I aou_callset_1K_229.vcf.gz \
---output chr20.vcf.gz
+[ "aou_callset_1K_223.vcf.gz", \
+  "aou_callset_1K_224.vcf.gz", \
+  "aou_callset_1K_225.vcf.gz", \
+  "aou_callset_1K_226.vcf.gz", \
+  "aou_callset_1K_227.vcf.gz", \
+  "aou_callset_1K_228.vcf.gz", \
+  "aou_callset_1K_229.vcf.gz"
 ```
-And create an index for that VCF:
+
+**output_basename** - The base name for the output files generated by the pipeline
+
+**chromosome** - The chromosome on which to run the analysis of Precision and Sensitivity. The default value for this is `chr20`. If it is set to `all` then the analysis will be run across *all* chromosomes.
+
+**sample_names** - A list of the sample names that are controls and that will be sued for the analysis. For every element on the list of sample names there must be a corresponding element on the list of `truth_vcfs`, `truth_vcf_indices`, and `truth_beds`.
+
 ```
-tabix chr20.vcf.gz
+[ "NA12878", \
+  "NA24385"
 ```
-Now create single sample gVCFs for the control samples; in this example the sample names for the controls are "BI_HG-002", "UW_HG-002" and "BI_HG-003":
+**truth_vcfs** - A list of the VCFs that contain the truth data used for analyzin the samples in `sample_names`.
```

Comment:
```suggestion
**truth_vcfs** - A list of the VCFs that contain the truth data used for analyzing the samples in `sample_names`.
```

Reply:
I am personally fine with us being too cool for the g, but in case it was a typo...